trello_items.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import secrets
+from card import Card
+
+key = secrets.apiKey
+token = secrets.apiToken
+
+def get_todo_cards_for_board():
+    """
+    Fetches all To Do cards from the board.
+
+    Returns:
+        list: The list of cards.
+    """
+
+    query = {
+        'key': key,
+        'token': token
+    }
+   
+    board_id = secrets.boardId
+    to_do_cards = []
+    url = "https://api.trello.com/1/boards/{}/cards".format(board_id)
+    cards = requests.request("GET", url, params=query)
+    for card in cards.json():
+        if (get_list_name(card['id']) == 'To Do'):
+            to_do_cards.append(Card(card['id'], card['pos'], card['name'], card['desc'], get_list_name(card['id'])))
```

Comment:
You already know the list name is 'To Do'. This line is inside an if which checks it.

As written, you're making an unnecessary extra HTTP request for every single card.

Reply:
Done